src/main/java/org/itsallcode/openfasttrace/Reporter.java

```diff
@@ -58,6 +58,14 @@
      */
     Reporter addInputs(final List<Path> inputs);
 
+    /**
+     * Set the list of artifact type to be ignored during import
+     * 
+     * @param ignoredArtifactTypes
+     *            list of ignored artifact types
+     */
+    Reporter ignoreArtifactTypes(List<String> ignoredArtifactTypes);
```

Comment:
This is duplicated in `Converter`. When we add new methods we should think of extracting a common interface.